### PR TITLE
refactor/ index.js in server folder add compression

### PR DESCRIPTION
**Pull request by mmarinm**
- refactor index.js in express, WebSocket and the server files inside of the server folder
- add compression package to improve express performance you can learn more about it at https://blog.fossasia.org/using-compression-middleware-in-nodeexpress/

### CODE REVIEW

File: server/websocket.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const server = require('./')
```
Comment (lfaudreejr) [inline server/websocket.js]: Is require('./') correct here?

Comment (mmarinm) [reply server/websocket.js]: when the name of the file is index.js you don't have to define the path for the file, path for the directory is enough

Comment (lfaudreejr) [reply server/websocket.js]: Ah. Ok, yeh I forgot about that. I like to be verbose though. I'm good with what you have there.

Comment (mmarinm) [reply server/websocket.js]: I am just used to writing paths like this. That's a good point about different personal styles. We should add pre-commit hooks with prettier and husky to make sure code styles are consistent

**Pull request merged**